Path2Communicates/UI.py

```diff
@@ -0,0 +1,101 @@
+import tkinter as tk
+import const
+from robot_message import RobotMessage
+
+
+class UI:
```

Comment:
Zmienić nazwę która będzie w sposób widoczny mówić że jest to UI do ścieżek i komunikatów

Reply:
Zmieńcie nazwę kklasy, nie tylko pliku